src/components/Rocket.jsx

```diff
@@ -1,7 +1,17 @@
 import React from 'react';
+import './rocket.css';
 
-export default function Rocket() {
+export default function Rocket(props) {
   return (
```

Comment:
- [ ] OPTIONAL... great job doing all these changes 👍 but Always use the arrow function instead of the normal function 💯

Reply:
@Haadiiii  Duly noted👍🏽. Thank you for the review